src/Components/FixableTabMenu/FixableTabMenu.js

```diff
@@ -0,0 +1,105 @@
+import { React, useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+
+function FixableTabMenu({ tabList, tabIndex, setTabIndex }) {
+  const [isFix, setIsFix] = useState(false);
+  const tabMenu = useRef();
+
+  useEffect(() => {
+    // const handleScroll = () => {
+    //   if (window.pageYOffset > tabMenu.current.offsetTop && !isFix) {
+    //     //
+    //     setIsFix(true);
+    //   } else if (window.pageYOffset <= tabMenu.current.offsetTop) {
+    //     //
+    //     setIsFix(false);
+    //   }
+    // };
+
+    const handleScroll = () => {
+      const nextIsFixed = window.pageYOffset > tabMenu.current.offsetTop;
+      const isDiff = isFix !== nextIsFixed;
+
+      if (isDiff) setIsFix(!isFix);
+    };
+
+    window.addEventListener('scroll', handleScroll);
+    return () => window.removeEventListener('scroll', handleScroll);
+  }, [isFix]);
+
+  return (
+    <>
+      <TabAreaHidden ref={ref => (tabMenu.current = ref)} />
```

Comment:
- ref 사용하실 때, (물론 지민님이 하신 방법도 한가지 방법이지만) 보통 다음과 같은 syntax 를 사용합니다. 
- 아래의 문법이 훨씬 간단하고, 가독성 측면에서도 좋아보이는데 어떻게 생각하시나요? ㅎㅎ
```suggestion
      <TabAreaHidden ref={tabMenu} />
```
- 선언하신 ref (tabMenu) 를 그냥 `ref` 라는 속성에 부여하시면 됩니다!

Reply:
코드 실행 중에 tabMenu이 undefined 오류 발생해서 변경해 두었는데, 지금 변경해서 실행해 보니 또 잘 동작하네요..! 
알려주신 문법으로 변경하였습니다!